config/crd/patches/webhook_in_integrationscenarios.yaml

```diff
@@ -7,10 +7,9 @@ spec:
   conversion:
     strategy: Webhook
     webhook:
+      conversionReviewVersions: ["v1alpha1", "v1beta1"]
```

Comment:
This file is similar with webhook_in_integrationtestscenarios.yaml. What is the reason to keep them both?

Reply:
Yes, I think it should be possible to remove this extra file without affecting anything.

Reply:
Removed the unused files, please take a look

Reply:
No reason at all, we can drop it, this file is not used.